### PR TITLE
Update README.md to have slack workspace annoucement at top of banner

### PR DESCRIPTION
**Pull request by scrawfor99**
Add slack workspace announcement at top of README for a couple weeks.

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
 [![](https://img.shields.io/codecov/c/gh/opensearch-project/security)](https://app.codecov.io/gh/opensearch-project/security) [![](https://img.shields.io/github/issues/opensearch-project/security/v2.4.0)](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3A"v2.4.0") [![](https://img.shields.io/github/issues/opensearch-project/security/v3.0.0)](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3A"v3.0.0")
 
+# ANNOUNCEMENT: The Slack workspace live! Please join the conversation by following the instructions at https://opensearch.org/slack.html.
```
Comment (DarshitChanpura) [inline README.md:4]: nit: `is live`.
Also, should we point them to security channel?

File: README.md
Comment (peternied) [inline README.md:1]: This is a little too intense for me personally, I'd recommend dialing the size down a bit.

**Review by peternied — COMMENTED** [review]
While you are announcing the slack instance could you add a badge for the slack?

File: README.md (hunk 1 above)
Comment (scrawfor99) [reply README.md:4]: Oops, sounds good.

**Comment by scrawfor99** [comment]
Closing because editing the markdown made a new PR at https://github.com/opensearch-project/security/pull/2653

**Pull request closed without merging**